packages/grid/_modules_/grid/hooks/features/export/seralizers/csvSeraliser.ts

```diff
@@ -1,9 +1,16 @@
 import { GridColumns, GridRowId, GridRowModel } from '../../../../models';
 
+const formatCellValue = (value) => {
+  if (typeof value === 'string') {
+    return value.includes(',') ? `"${value}"` : value;
```

Comment:
what if the value contains dbl quote?

Reply:
it will still work as expected, the problem comes when you have a structure like this `value"value,value"value`

Reply:
but if you don't escape the double quote it will be parsed as another col if it contains the ','

ie cell value: 'abc , "quote, from a great guy" is easy '

Reply:
I will a do PR that will add a getCellValue on apiref

Reply:
The case is now handled